soc/riscv/riscv-ite/common/policy.c

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2022 ITE Corporation. All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/pm/policy.h>
+#include <soc.h>
+
+__weak const struct pm_state_info *pm_policy_next_state(uint8_t cpu, int32_t ticks)
```

Comment:
This looks like a copy of the default PM policy minus residency time checks. What is its purpose?

Reply:
Yes, we don't need to check residency time, because we have a 15s interval setting to block enter the power policy when uart wakeup.